tox.ini

```diff
@@ -6,13 +6,16 @@ skipsdist = True
 [testenv]
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/test-requirements.txt
+commands = find {toxinidir} -type f -name "*.py[c|o]" -delete
 
 [testenv:black]
+basepython = python3
```

Comment:
should we consider making basepython default in testenv?

Reply:
I wanted to leave that open just in case you were in need for that *not* to happen because of the way zuul is set up for collections. If that's ok for you I'll set it up globally.

Reply:
Yah, looking at python3 migration for openstack, that is what they did.

https://opendev.org/openstack/nova/src/branch/master/tox.ini#L10

We should do the same

Reply:
Yah, lets default to python3, python2 is EOL now